### PR TITLE
PMM-3478: runs_on_node_id to pmm_agent_id

### PR DESCRIPTION
**Pull request by DexterHD**
I have made stupid mistakes in commit messages. 😞  It is all about PMM-3478, not for PMM-3548

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ required = [
 # TODO switch to tags before PMM 2.0 release
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-3478-runs_on_node_id-to-pmm_agent_id"
```
Comment (AlekSi) [inline Gopkg.toml]: Don't forget to change it back before merging.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://travis-ci.org/percona/pmm-managed.svg)](https://travis-ci.org/percona/pmm-managed)
 [![Go Report Card](https://goreportcard.com/badge/github.com/percona/pmm-managed)](https://goreportcard.com/report/github.com/percona/pmm-managed)
 [![CLA assistant](https://cla-assistant.io/readme/badge/percona/pmm-managed)](https://cla-assistant.io/percona/pmm-managed)
+[![codecov.io Code Coverage](https://img.shields.io/codecov/c/github/percona/pmm-managed.svg?maxAge=2592000)](https://codecov.io/github/percona/pmm-managed?branch=PMM-2.0)
```
Comment (AlekSi) [inline README.md]: https://codecov.io/gh/percona/pmm-managed/branch/PMM-2.0/graph/badge.svg

File: models/agent.go [hunk 3]
```diff
@@ -55,11 +55,10 @@ func AgentsForNode(q *reform.Querier, nodeID string) ([]*Agent, error) {
 	return res, nil
 }
 
-// AgentsRunningOnNode returns all Agents running on Node.
-// TODO Remove after https://jira.percona.com/browse/PMM-3478.
-func AgentsRunningOnNode(q *reform.Querier, nodeID string) ([]*Agent, error) {
-	tail := fmt.Sprintf("WHERE runs_on_node_id = %s ORDER BY agent_id", q.Placeholder(1)) //nolint:gosec
-	structs, err := q.SelectAllFrom(AgentTable, tail, nodeID)
+// AgentsRunningByPMMAgent returns all Agents running by PMMAgent
```
Comment (AlekSi) [inline models/agent.go]: `.`

**Comment by AlekSi** [comment]
Squash commits together with correct ticket number when everything else is ready.

**Comment by DexterHD** [comment]
@AlekSi All fixed, include change branch back to "PMM-2.0" Waiting for CI.

**Pull request merged**